R/retry.R

```diff
@@ -92,8 +94,8 @@ backoff_full_jitter <- function(i, resp, pause_base = 1, pause_cap = 60,
       status <- status_code(resp)
     }
     if (status == 429) {
-      length_429 <- as.numeric(resp$headers[["retry-after"]])
-      if (!is.null(length_429) && length_429 >= pause_min) length <- length_429
+      retry_after <- resp$headers[["retry-after"]]
+      if (!is.null(retry_after)) length <- max(pause_min, as.numeric(retry_after))
```

Comment:
Can you please put the condition code inside of `{}` (indented like the rest of the code)